### DISCUSSION TITLE
遗漏我2枚奇亚币

### DISCUSSION THREAD

**Discussion by longteng023** [Chia-Network/chia-blockchain#5099] [Support]
我收到奇亚奖励，而 我的钱包并没有增加。4月29号币钱包里面有2枚，5月8号和5月9号的币没有。不知道为什么
<HTML><BODY>

收入 | xch1lszr8uet43um7hrz4c9luqa9sejyxplqm3g9g789nyt5ymrc8hlsnm098y | 05/13/2021 下午11:17:39 | 区块高度确认 274544 | 0.0000000001 | 0
-- | -- | -- | -- | -- | --
收入 | xch1lszr8uet43um7hrz4c9luqa9sejyxplqm3g9g789nyt5ymrc8hlsnm098y | 05/09/2021 上午8:55:11 | 区块高度确认 252811 | 0.75 | 0
收入 | xch1nd86002lt5f95afe2vwpr9q0h50parm4uttwsm84n2j5p65rfzfs6w69xq | 05/08/2021 上午8:55:49 | 区块高度确认 247748 | 1.63 | 0
收入 | xch1x74usf9upy679qaf3m6kxgf23e0ks9tt6x2hnmxr0p7tjw8s5mkq4derw3 | 04/29/2021 上午6:29:51 | 区块高度确认 181925 | 1.75 | 0
收入 | xch1x74usf9upy679qaf3m6kxgf23e0ks9tt6x2hnmxr0p7tjw8s5mkq4derw3 | 04/29/2021 上午6:29:51 | 区块高度确认 181925 | 0.25 | 0

</BODY></HTML>

**Comment by uwerui** [comment]
The receiving address is different

**Reply by playzhou** [reply]
仔细看你两次的钱包地址不一样，是不是找别人给你P的盘，然后 P的不是你的钱包啊。。。。

**Reply by Heuer212** [reply]
自己的钱包地址也是会变的，我全自己p的也遇到了这种情况，不知道咋回事

**Reply by playzhou** [reply]
> 自己的钱包地址也是会变的，我全自己p的也遇到了这种情况，不知道咋回事

.... 啊？那怎么办？

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
